plugins/scripting/scripting-ide-services-test/test/org/jetbrains/kotlin/scripting/ide_services/ReplTest.kt

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.scripting.ide_services
+
+import junit.framework.TestCase
+import kotlinx.coroutines.runBlocking
+import org.jetbrains.kotlin.scripting.ide_services.compiler.KJvmReplCompilerImpl
+import org.jetbrains.kotlin.scripting.ide_services.evaluator.KJvmReplEvaluatorImpl
+import org.jetbrains.kotlin.scripting.ide_services.test_util.captureOut
+import org.jetbrains.kotlin.scripting.ide_services.test_util.simpleScriptCompilationConfiguration
+import org.jetbrains.kotlin.scripting.ide_services.test_util.simpleScriptEvaluationConfiguration
+import org.junit.Assert
+import org.junit.Test
+import kotlin.script.experimental.api.*
+import kotlin.script.experimental.host.toScriptSource
+import kotlin.script.experimental.util.isErrorResult
+import kotlin.script.experimental.util.isUnitResult
+import kotlin.script.experimental.util.isValueResult
+
+class ReplTest : TestCase() {
```

Comment:
It seems that these tests are ide-services-independent. If so, I'd prefer them to be moved to the jvm-host-tests

Reply:
Moved